technical-paper/sections/05-contents.adoc

```diff
@@ -0,0 +1,18 @@
+
+== Specific contents: could be introduction or something more substantive
+// [ADD TEXT.]
+
+.Example of a table
+[cols="2*",options="header"]
+|===
+|Number |Observation
+
+|1 |Sample table
+|2 |Table header (1st row) uses style Table_head
+|3 |Other table rows use style Table_text
+|4 |Heading with table number uses style "Table_No & title"
+|===
```

Comment:
Add empty line before table end (otherwise it is considered the table "footer")

Reply:
Change applied.